core/src/components/refresher-content/refresher-content.tsx

```diff
@@ -52,18 +52,18 @@ export class RefresherContent implements ComponentInterface {
 
   componentWillLoad() {
     if (this.pullingIcon === undefined) {
-      const mode = getIonMode(this);
+      const platform = getIonPlatform(this);
       const overflowRefresher = (this.el.style as any).webkitOverflowScrolling !== undefined ? 'lines' : arrowDown;
       this.pullingIcon = config.get(
         'refreshingIcon',
-        mode === 'ios' && isPlatform('mobile') ? config.get('spinner', overflowRefresher) : 'circular'
+        platform === 'ios' && isPlatform('mobile') ? config.get('spinner', overflowRefresher) : 'circular'
```

Comment:
Optional: I wonder if the `isPlatform` util should be renamed? It looks confusing to use it and `getIonPlatform` at the same time like this, since the two are actually referring to different things.

Reply:
I'd prefer not to name `isPlatform` since that is a public API. What if we changed `getIonPlatform` to `getIonCapability`? 

IMO our internal usage of the term "mode" is a bit loaded now. It might make sense to do the following:

`getIonMode` --> `getIonStyle`: This returns the visual style that a component is running in.
`getIonPlatform` --> `getIonCapability`: This returns the platform that a component is running in.

Reply:
Actually `getIonStyle` may collide with that internal `ionStyle` event we use in some components. Maybe `getIonMode` --> `getIonVisual` or `getIonStylesheet`?

Reply:
`getIonStylesheet` sounds great 👍 For `getIonPlatform`, maybe something like `getIonBehaviors`? I feel like that better captures how we're cherry-picking certain pieces of functionality. Super bikeshedding though, `getIonCapability` also works for me.